aut_docs/Installation_Setup.md

```diff
@@ -0,0 +1,103 @@
+## Steps to run tests:
+
+Following 4 points are to be followed only once to setup repository.
```

Comment:
"A one-time setup requires the following four steps"

Prefer words intext over numerals.

Reply:
Done :+1: